english/explore-outside/talk-child/talk-more/nice-talk/help-child/keep-looking/enter-cave/follow-person/follow-person.md

```diff
@@ -1,4 +1,14 @@
 You use all your strength to swim toward the light. The person ahead swam out of sight. Can you make it?
 Yes! When your strength was almost spent you realized that it was getting shallow and you could stand up.
 There is an air space above your head! You can breathe!
-Taking in great heaving gulps of sweet air (well, kind of damp cave air) you...
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.